### DISCUSSION TITLE
React

### DISCUSSION THREAD

**Discussion by liyanage1234** [generate-all/generate-all-in-one#12] [Q&A] [answered]
How can I start the development server in React ?

**Comment by Heshan-Lahiru** [comment, marked as answer]
npm start

**Comment by Dhruv6397** [comment]
![Screenshot 2024-10-02 151310](https://github.com/user-attachments/assets/7919d6d6-f272-4c68-a313-95f888764006)


make sure u must have above code written in script .
then u can start your server, if you have properly implemented your index.js/server.js in your project

To start from nodemon enter following command
npm run dev

To start from node enter following command
npm run start
